test/unit/file-handlers/html-test.js

```diff
@@ -321,5 +322,53 @@ describe('Html handler', function () {
 			}).catch(done);
 		});
 
+		it('should not prettifyUrls by default', function(done) {
+			var loadStub = sinon.stub(scraper, 'loadResource');
+			loadStub.onFirstCall().returns(Promise.resolve(new Resource('http://example.com/other-page/index.html', 'other-page/index.html')));
+
+			var html = ' \
+				<html lang="en"> \
+				<head></head> \
+				<body><a href="other-page/index.html">Other page</a></body> \
+				</html>\
+			';
+			var po = new Resource('http://example.com', 'index.html');
+			po.setText(html);
+
+			return loadHtml(scraper, po).then(function () {
+				var text = po.getText();
+				text.should.containEql('a href="other-page/index.html"');
+				done();
+			}).catch(done);
+		});
+
+		it('should not prettifyUrls if specified', function(done) {
```

Comment:
seems `not` is not needed here 
it should be `should prettifyUrls if specified`